specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -1308,7 +1308,9 @@
           "description": "Type of managed service identity.",
           "enum": [
             "SystemAssigned",
-            "UserAssigned"
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
```

Comment:
Why is `UserAssigned` repeated?

Reply:
The values "UserAssigned" and "SystemAssigned, UserAssigned" are distinct enum values. You can see another example on Compute's swagger here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json#L5867